src/ankit/Bank.java

```diff
@@ -0,0 +1,97 @@
+package ankit;
+
+public class Bank {
+	String custName;
+	int balance;
+	int indCreditCount = 0;
+	int indDebitCount = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.